scripts/python/services/test_GithubService.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+from unittest.mock import call, MagicMock, patch
+
+from scripts.python.services.GithubService import GithubService
+
+ORGANISATION_NAME = "moj-analytical-services"
+
+
+class TestGithubService(unittest.TestCase):
+    mock_github = None
+
+    github_service = None
+
+    def setUp(self):
+        self.mock_github = MagicMock()
+        self.patcher = patch(target="github.Github.__new__",
+                             return_value=self.mock_github, autospec=True)
+        self.patcher.start()
+        self.addCleanup(self.patcher.stop)
+
+        self.github_service = GithubService("", ORGANISATION_NAME)
+
+    def tearDown(self):
+        self.patcher.stop()
+
+    def test_innit_sets_up_class(self):
```

Comment:
should this be innit or init?

Reply:
Should be `init`! That's my horrible spelling and grammar side creeping into the code, innit 😂